libs/agno/agno/os/app.py

```diff
@@ -557,7 +567,7 @@ def _get_knowledge_config(self) -> KnowledgeConfig:
                     DatabaseConfig(
                         db_id=db_id,
                         domain_config=KnowledgeDomainConfig(
-                            display_name="Knowledge" if not multiple_knowledge_dbs else "Knowledge in database " + db_id
+                            display_name="Knowledge" if not multiple_knowledge_dbs else db_id
```

Comment:
Not just db_id?

Reply:
Yes it will db_id, missed for knowledge , thanks for catching it

Reply:
Updated it @dirkbrnd